apps/microsoft-teams/frontend/src/components/config/ContentTypeSearch/ContentTypeSearch.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useMemo } from 'react';
+import { Box, IconButton, TextInput } from '@contentful/f36-components';
+import { SearchIcon } from '@contentful/f36-icons';
+import { contentTypeSelection } from '@constants/configCopy';
+import { debounce } from 'lodash';
+
+type Props = {
+  handleChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const ContentTypeSearch = ({ handleChange }: Props) => {
```

Comment:
Abstract this component to be used for both content type searching and teams channel searching.

- [ ]